docs/cep/reference/extensions/execution/fx.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Function Worker
+---
+
+The `fx` extension provides access to Function Worker.
+
+Function worker allows you to execute code in response to events/requests without the complex infrastructure typically associated with building and launching microservices applications. It supports multiple function worker types:
+* Akamai EdgeWorker. It enables developers to create and deploy microservices across more than a quarter of a million edge servers deployed around the globe.
```

Comment:
This sentence seems like marketing copy more then technical documentation. Consider linking to the documentation for Acamai EdgeWorkers, if any exists, rather than this.

Reply:
@diana-macrometa  - I will suggest to remove explanation of types function workers because it doesn't care which type it is.